ophyd/sim.py

```diff
@@ -784,36 +786,37 @@ def flyer_worker():
             st.set_finished()
 
         threading.Thread(target=flyer_worker, daemon=True).start()
-
-        return st
+        kickoff_st = DeviceStatus(device=self)
+        kickoff_st.set_finished()
```

Comment:
Does it mean it will return the `done` status immediately? Is it what we need?

Reply:
Yes.  I guess we could be very fancy and do this in at the top of the thread, but there is nothing we are waiting on here for the simulated device to report "I have started flying!".

The goal is to avoid the following:

1. kick off flyer
2. start moving motor
3. move through data range
4. flyer finishing arming itself and starts collecting data

so kickoff is an async thing so we can do

1. kick off flyer1
2. kickoff flyer2
3. wait for both to report they are read to take data
4. start moving motor
5. ????
6. profit!

Reply:
That makes sense to me! Thanks for the details.